### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by liondadev**
Remove double prefixes with the new xAdmin pr, and use xAdmin.Core.Msg instead of xWarn.msg because it looks better. This also stops it from erroring out when using it on sqlite (specifically when warning someone).

Might require https://github.com/TheXYZNetwork/xAdmin/pull/34

### CODE REVIEW

File: lua/xwarn/commands/sv_warn.lua
Comment (Creekie1337) [inline lua/xwarn/commands/sv_warn.lua]: Ideally we want to keep the warning ID in the actual message, it makes it easy to delete an accidental warn without having to go through someone's warns.

Could just make it a config option as some servers might not need it.

File: lua/xwarn/commands/sv_warn.lua
Comment (Creekie1337) [inline lua/xwarn/commands/sv_warn.lua]: `user` is nil here, guessing you meant `admin`

File: lua/xwarn/commands/sv_warn.lua
Comment (Creekie1337) [inline lua/xwarn/commands/sv_warn.lua]: `user` is nil here, guessing you meant `admin`

File: lua/xwarn/commands/sv_warn.lua
Comment (Creekie1337) [inline lua/xwarn/commands/sv_warn.lua]: `user` is nil here, guessing you meant `admin`

**Review by Creekie1337 — CHANGES_REQUESTED** [review]
Just a few small issues, otherwise LGTM.

**Comment by liondadev** [comment]
Should all be fixed in the latest commit :)

**Review by Creekie1337 — APPROVED** [review]
LGTM. Approved for merge, once the xAdmin one is merged I'll merge this.

**Pull request merged**
